internal/operation/spec.go

```diff
@@ -31,6 +31,7 @@ type Spec struct {
 	Edges      []Edge                  `json:"edges"`
 	Resources  flux.ResourceManagement `json:"resources"`
 	Now        time.Time               `json:"now"`
+	Conflicts  bool
```

Comment:
Can you add a comment here that describes what `Conflicts` indicates? I know this is temporary code, but I would like to understand.

Reply:
Done.